CondFormats/GeometryObjects/src/classes_def.xml

```diff
@@ -51,7 +51,7 @@
  <class name="PMTDParameters::Item"/>
  <class name="std::vector<PMTDParameters::Item>"/>
 
- <class name="HcalParameters" class_version="0"> 
+ <class name="HcalParameters" class_version="1"> 
```

Comment:
If I'm not wrong, a class_version == 0 tells ROOT not to store that object.: do we really want to update this?
@Dr15Jones @makortel

Reply:
class_version of 0, 1 (and sometimes 2) have special meaning to ROOT. I'm fairly certain these should all stay at 0.